Source/SwiftyJSON/SwiftyJSON.swift

```diff
@@ -301,14 +301,16 @@ private func resolveContent(for jsonObject: Any) -> Content {
         return .dictionary(dictionary.mapValues(resolveContent))
     case let array as [Any]:
         return .array(array.map(resolveContent))
-    case nil:
-        return .null
-    case is NSNull:
-        return .null
     case let string as String:
         return .string(string)
     case let number as NSNumber:
         return number.isBool ? .bool(number.boolValue) : .number(number)
+    case nil:
+        return .null
+    case is NSNull:
+        return .null
+    case is AnyOptional:
```

Comment:
This would match any optional which is not previously matched, even if it's not `nil`